doc/manual/parallel-computing.rst

```diff
@@ -451,36 +550,33 @@ preemptively. This means context switches only occur at well-defined
 points: in this case, when :func:`remotecall_fetch` is called.
 
 
-Channels
```

Comment:
I think it is important retain this section and differentiate between `Channels` which are used for inter-task communication and `RemoteChannels`/`Futures` which are used for inter-process communication.